app/views/documentation/v1.scala.html

```diff
@@ -14,7 +14,7 @@
         @for(version <- context.version) {
             <form id="search" method="get" action="//www.google.com/search">
                 <input type="search" name="q" placeholder="Search @version docs" results="10">
-                <input type="hidden" name="sitesearch" value="@req.host@reverseRouter.home(context.alternateLang, version.name)">
+                <input type="hidden" name="sitesearch" value="site:playframework.com@reverseRouter.home(context.alternateLang, version.name)">
```

Comment:
@cies Please remove the `site:`, it gets added from google automatically. Actually having it breaks the google search.

Reply:
Done!